x-pack/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -111,7 +110,16 @@ export class TransactionActionMenu extends React.Component<Props, State> {
         hash: `/link-to/host-logs/${hostName}`,
         query: { time }
       },
-
+      {
+        icon: 'loggingApp',
+        label: i18n.translate(
+          'xpack.apm.transactionActionMenu.showTraceLogsLinkLabel',
+          { defaultMessage: 'Show trace logs' }
+        ),
+        target: traceId,
+        hash: `/link-to/logs`,
+        query: { time, filter: `trace.id:${traceId}` }
```

Comment:
I'm not seeing this here, but wanted to mention it because I got burned by this before:

Characters like `:` in the URL might be strictly encoded by the `querystring` module, which is seems to be used here. I have run into situations where this caused duplicate history entries due to angular undoing the encoding of certain characters. I previously prevented that by using the encoding functions provided by `ui/utils/query_string` instead of those from `querystring`.

Reply:
Thanks, we'll look into this -- we already handle this issue with the % character, but maybe we can consolidate into one solution.